recipes/alchemiscale/meta.yaml

```diff
@@ -0,0 +1,102 @@
+{% set name = "alchemiscale" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: alchemiscale
+  version: {{ version }}
+
+source:
+  url: https://github.com/openforcefield/alchemiscale/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: d46f372adcaa257eb909587f668e791915188795b60db0c11bbfd6bfe9ac1a84
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
B/c the outputs are noarch this will be installable on Windows regardless of this skip. What is the reason for it? If it is just a missing dependency, then it is fine to remove it and let the solver deal with it. A failure in that case is not a blocker for merging.

Reply:
Can do! I've gotten advice both ways and for noarch I also like to have the solver deal with it.